multimodal-search-tirg/pods/encode-tirg-multimodal.yml

```diff
@@ -1,5 +1,7 @@
-!DummyTirgMultimodalEncoder
+!DummyEncoder
```

Comment:
the `multimodal` encoder I would create a different `DummyMultiModalEncoder` that actually `asserts` that the first modality is `visual` and the second one is `textual` ? or however we want to do.

Reply:
We still can't work without the `MultiModalDriver`

Reply:
Yes I wanted to ask you about it but got the answer. Also, I think we should move the 'DummyEncoder` to our hub as we will need it in future + its good for dry run of flow which might be a feature of dashboard in the future.

Reply:
we will see in the future, for now, it is just a temporal thing here.